### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -18076,6 +18080,7 @@ mmatching->matching
 mmbers->members
 mmnemonic->mnemonic
 mnay->many
+moast->most
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:18083]: Suggest moat too

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -30598,6 +30605,7 @@ wnat->want, what,
 wnated->wanted
 wnating->wanting
 wnats->wants
+wnen->when
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:30608]: Suggest Wen name too.

**Review by peternewman — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:30608]: conflicts with https://github.com/luzpaz/codespell/blob/d8c56c34fa8641b75dbcd6341778af626059cc2c/codespell_lib/data/dictionary_names.txt#L11

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Added in c0b2155

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Added anyway in 8088c43

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Fixed in 036cf1db2f2cc62e4c440527807d91a5382f07b8

**Review by peternewman — APPROVED** [review]

**Pull request merged**